### PR TITLE
Add polar stereo

### PR DESCRIPTION
**Pull request by mjhossen**
Modified the code ``` create_plots.py and map_tools.py``` to enable us to plot figures over polar region. I plot the following figures using ```testIodaObsSpaceAmsuaN19.yaml```

Screen Shot 2022-11-02 at 11 31 49 AM

### CODE REVIEW

**Review by danholdaway — DISMISSED** [review]

File: src/emcpy/plots/create_plots.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def _map_scatter(self, plotobj, ax):
 
             cs = ax.scatter(plotobj.longitude, plotobj.latitude,
                             s=plotobj.markersize, **inputs,
-                            transform=self.projection.projection)
+                            transform=self.projection.transform)
```
Comment (CoryMartin-NOAA) [inline src/emcpy/plots/create_plots.py:393]: want @kevindougherty-noaa to confirm this still works as intended with this change

Comment (CoryMartin-NOAA) [reply src/emcpy/plots/create_plots.py:393]: Oh I see what this is doing now, it should be fine.

**Review by CoryMartin-NOAA — APPROVED** [review]
Approval contingent on CI tests passing.

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]
@mjhossen can you fix the coding norms issue?

File: src/emcpy/plots/create_plots.py (hunk 1 above)
Comment (kevindougherty-noaa) [reply src/emcpy/plots/create_plots.py:393]: Yes, transform for NP/SP stereographic projections have to be PlateCarree().

**Review by kevindougherty-noaa — COMMENTED** [review]
Looks good, thank you for making the changes. Do mind sharing the yaml that produced the above figure just so I fully understand your process before I approve?

**Comment by mjhossen** [comment]
> Looks good, thank you for making the changes. Do mind sharing the yaml that produced the above figure just so I fully understand your process before I approve?

Yaml file is pushed to  https://github.com/JCSDA-internal/eva/pull/61

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by kevindougherty-noaa — APPROVED** [review]
Thanks again for making these changes. I understand what you're doing now, great work.

**Pull request merged**